pkg/controller/options.go

```diff
@@ -33,4 +36,13 @@ type Options struct {
 	// resource. Setting this enables External Secret Stores for the controller
 	// by adding connection.DetailsManager as a ConnectionPublisher.
 	SecretStoreConfigGVK *schema.GroupVersionKind
+
+	// ESSOptions for External Secret Stores.
+	ESSOptions *ESSOptions
+}
+
+// ESSOptions for External Secret Stores.
+type ESSOptions struct {
+	TLSConfig     *tls.Config
```

Comment:
How are we going to set the `controller.Options.ESSOptions.TLSConfig`? In the current implementation, the provider will panic if the `--enable-external-secret-stores` command-line option is passed to the provider but the `ESSOptions` is not set.

Reply:
We're passing the certificate path through env variable and if it's not set, we're not loading the config.

Reply:
Thanks for the reference PR @ezgidemirel!